src/main/java/org/kiegroup/kogibot/listeners/pr/ideas

```diff
@@ -0,0 +1,9 @@
+// check the PR patterns, look for a configuration file on the target repository
+// e.g. .kogibot.yaml
+// it can contain the desire pattern for:
+// 1 - Tittle pattern
+// 2 - Description patter
+// 3 - PR commit message
+// Bot will automatically not approve this PR until this condition is not satisfied.
+// Use the util.PullRequestPatternAnalyzer class to implement it.
```

Comment:
maybe add a check to verify if the commits are signed, maybe a flag in the config

Reply:
This is for later implementation. Let's start small before adding new functionatlities. My goal is primarly to gather what you and Davide did, add some tests and correct and then we could launch the bot on one or 2 repos before deploying to all kogito repo. wdyt ?
the PR analysis would be an improvement anyway. I don't want to add too much thing at first.